tests/unit/utils/aws/test_aws_responses.py

```diff
@@ -0,0 +1,27 @@
+import xml.etree.ElementTree as ET
+
+from localstack.utils.aws.aws_responses import to_xml
+
+
+def test_to_xml():
```

Comment:
could we add a couple of error case tests to document the behavior? for example passing non-dict types, None types, or dicts with nested classes.

Reply:
In general these are internal serialization functions, not something that is user-facing, so the type contract already "forces" you to use a dict here but I've added a few more edge cases in the tests. :+1: